src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsEmbeddedFractures.cpp

```diff
@@ -33,8 +33,8 @@ namespace geosx
 using namespace dataRepository;
 using namespace constitutive;
 
-SinglePhasePoromechanicsSolverEmbeddedFractures::SinglePhasePoromechanicsSolverEmbeddedFractures( const std::string & name,
-                                                                                                  Group * const parent ):
+SinglePhasePoromechanicsEmbeddedFractures::SinglePhasePoromechanicsEmbeddedFractures( const std::string & name,
+                                                                                      Group * const parent ):
```

Comment:
Great that you removed `Solver` in the class name, we should keep propagating that to other coupled solvers.